scripts/manhole-cli

```diff
@@ -27,7 +27,7 @@ for sig, num in vars(signal).items():
         SIG_NUMBERS.add(num)
 
 
-def parse_pid(value, regex=re.compile(r'^(/tmp/manhole-)?(?P<pid>\d+)$')):
+def parse_pid(value, regex=re.compile(r'^(/tmp(/.*?)?/manhole-)?(?P<pid>\d+)$')):
```

Comment:
Maybe we could even have `^(.*/manhole-)?(?P<pid>\d+)$`? In retrospect, I haven't considered that people would use manhole with other kind of paths when I wrote that regex ...